WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailCommentsTableViewDelegate.swift

```diff
@@ -114,13 +114,31 @@ class ReaderDetailCommentsTableViewDelegate: NSObject, UITableViewDataSource, UI
         return header
     }
 
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard section == 0, JetpackBrandingVisibility.all.enabled else {
+            return nil
+        }
+        return JetpackButton.makeBadgeView(bottomPadding: 10)
```

Comment:
since there is a `Constants` type, can we put this magic number in there?

Reply:
Good idea - added in 2653864e929c06397e320830f700eb2092777f63.